src/components/tools/RulerTool.vue

```diff
@@ -1,44 +1,19 @@
 <template>
   <div class="overlay-no-events">
     <svg class="overlay-no-events">
```

Comment:
I'm leaving it in RulerTool so that we don't unnecessarily create a lot of svg roots.

Inside RulerTool:
```
<svg>
  <g /> <!-- from RulerWidget2D -->
  <g /> <!-- from RulerWidget2D -->
 ...
```

If the svg were inside RulerWidget2D, it would look like the following:

```
<svg>
  <g />
</svg>
<svg>
  <g />
</svg>
...
```